changelogs/unreleased/gh-8051-set-box-cfg-thru-env.md

```diff
@@ -0,0 +1,4 @@
+## feature/box/cfg
```

Comment:
> box: allow setting box.cfg values via env vars

> Title: Set box.cfg options via env vars

The titles are confusing. This ability was implemented in 1b330121ec7dafcc0fc6ab63279e0dc52f9f947c. Next, several mappings (non-array tables) were added as box.cfg option. The feature was not supported only for them.

Reply:
Rephrased it to this:
`box: allow to set box.cfg table values via env var`
It is not perfect, but I don't see how to fit all specifics into 50 symbols.